### ISSUE TITLE
(noob-ish Fedora issue) Linux bin won't run without some fenagling with Go dev env prerequisites.

### ISSUE THREAD

**Issue by quix-cafe** [satisfactorymodding/SatisfactoryModManager#187]
Basically a mirror of https://github.com/projectdiscovery/subfinder/issues/1055 -- I'm a Pioneer, not a Programmer!

But I only got the "SatisfactoryModManager-Linux" bin to run on my vanilla Fedora 39 box only through some hours of fenagling with setting up dependencies in Go due to an issue with "aead". 

I know it's an alpha release but there *could* be a bit of documentation for getting it moving on Linux boxes, as it's (guessing), no longer an AppImage? Or maybe it's easy on non-Fedora distros.

Newb issues, but I just wanted to play some modded Satisfactory, so maybe this is helpful or maybe everybody's rolling their eyes at me lol. 

Much love y'all!

**Comment by mxqxxx** [comment]
To clarify, I now have it running, but can't reasonably provide "how" as I was basically throwing spaghetti at the walls. 

AND IN THAT SENSE, a beautiful update, working very well, and I'm excited to keep-on-a-goin with this install :)

**Comment by mircearoata** [comment]
Are you sure it's the same issue as the one you linked? That seems to be specific to running `go install <package>`, which is not how SMM is meant to be installed. By the time the app is compiled, there is nothing related to imports that could error, it's all just native code.

As far as I know (haven't looked too much into it), the only runtime dependency of SMM (that is not common with most other apps) is WebKitGTK.

I can try looking into generating an AppImage once again, but it previously was just how electron-builder packages the app, given it needs its chromium. Wails does not provide this currently (it will in Wails v3), so I'll have to look into the AppImage creation tools, and which dependency libraries should and shouldn't be in an AppImage.

**Comment by mxqxxx** [comment]
I'm *not* sure that the issue is *specific* to the one I linked -- only that my kerfuffling through those broken dependencies while working on compiling from source is what allowed the binary download to go from not executing to executing -- so my *hunch* is that having a working Golang development environment -- over the hurdles of the broken 'aead' dependency -- is necessary for the binary to execute.

or not? it has no file extension, and all the documentation is about compiling from source. trying to execute it simply kept posting the 'aead' dependency errors (or nothing) until I solved that. so my only 'qualm', or moreso suggestion for easing adoption by other Fedora users, is a bit more documentation for users going from a non-dev environment. 

I'll likely do a fresh install on my other box and try the whole rigmarole again soon -- and try to get some debug info to you if relevant. 

This issue still feels 'active' though without at least a note on how to use the Linux binary -- am I missing something simple?
